packages/bootstrap/scss/grid/_variables.scss

```diff
@@ -24,6 +24,8 @@ $grid-cell-padding-y: $grid-padding-y !default;
 $grid-cell-horizontal-border-width: 0px !default;
 $grid-cell-vertical-border-width: 1px !default;
 
+$grid-cell-checkbox-margin-y: calc( #{$cell-padding-y} - (#{$button-calc-size} - #{$line-height-em}) / 2 ) / 1.25  !default; // $grid-command-cell-padding-y / 1.25
```

Comment:
This is not a valid value. Here, we are assuming that the `postcss-calc` will remove the `calc`, however, if any of these variables change, and the `calc` is not removed, we will have an invalid output of the CSS.